src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+import './calculator.css';
+
+const Calculator = () => (
+  <section>
+    <div className="container">
+      <div className="display">
+        <div className="display-result">0</div>
+      </div>
+      <div className="all-buttons">
+        <button type="button" className="button clear-all">AC</button>
+        <button type="button" className="button last-entity-clear">+/-</button>
+        <button type="button" className="button operation">%</button>
+        <button type="button" className="button operation">/</button>
```

Comment:
- [ ] Please model your styling according to the provided template 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/165387700-9da47ad3-ddf9-4a08-a875-c8c397dd8b06.png)
### Given Design
![image](https://user-images.githubusercontent.com/46839548/165387544-738c0e85-96ee-45c4-99b1-88a81fc01ba6.png)